### PR TITLE
Feature/albums

### PR DESCRIPTION
**Pull request by larkinds**
Fully functioning albums sidebar with tracks that play/add-to-queue

### CODE REVIEW

File: src/albums.ts
Comment (jkmartindale) [inline src/albums.ts]: Gimme that end of file line break :'(
```suggestion
}

```

File: src/albums.ts
Comment (jkmartindale) [inline src/albums.ts]: ?

File: src/albums.ts
Comment (jkmartindale) [inline src/albums.ts]:
```suggestion
    super(data.name ?? data.album.name);
```

File: src/albums.ts
Comment (jkmartindale) [inline src/albums.ts]:
```suggestion
      this.description = this.artists;
```

**Review by jkmartindale — CHANGES_REQUESTED** [review]
Just wanna see the track labels fixed like with playlists

Since you branched off `feature/playlist` instead of `staging` (totally understand how easy it is to accidentally branch off another feature branch), we'll need to wait until #30 gets merged in, and then very carefully resolve merge conflicts coming from this branch's copy of the work in `feature/playlist`. Should probably be merged in with **Squash and Merge**.

File: src/albums.ts
Comment (larkinds) [reply src/albums.ts]: Yeahhhhhh 😋

**Pull request merged**
